src/pages/Landing/Landing.jsx

```diff
@@ -1,20 +1,100 @@
+/* eslint-disable prettier/prettier */
 import React from 'react'
 
 import NavBar from './components/NavBar/NavBar'
 import Footer from './components/Footer/Footer'
 
 import styles from './Landing.module.scss'
 import Hero from './components/Hero/Hero'
+import Course from './components/Course/Course'
+import Places from './components/Places/Places'
+import App from './components/App/App'
+import SimpleSwiper from './components/Sliders/Sliders'
 
```

Comment:
can we enable and use [sort-imports](https://marketplace.visualstudio.com/items?itemName=amatiasq.sort-imports#:~:text=Automatically%20sorts%20imports%20on%20save,Enter%20Sort%20imports)